docs/05_project_metrics.md

```diff
@@ -1,20 +1,117 @@
 # Project metrics definitions
 
-## Build Result
-Project build result - successful or failed. 
+## Build Status Metric
+#### Description 
+A build status metric is the result of the latest build. It can be successful, failed, or unknown.
+* A successful build result indicates the success of all necessary checks, such as tests, code style analysis, builds, 
+and others.   
+* In any other cases the build result is unknown.
+* An unknown build status indicates an absence of builds to display a build result.
+#### Source
+The information about the build result comes from the CI system with the other information about other builds.  
+#### Date ranges
+The build status includes the status of the latest build.
+#### Appearance
+* A successful build result is a checkmark on a green background. 
+* A failed build result is a cross on a red background. 
+* An unknown build result is a dash on a gray background.   
 
-## Performance
-Average build duration (without queue time) of successful builds.
+![Build Statuses (Dark theme)](https://github.com/platform-platform/monorepo/blob/update_project_metrics_document/docs/images/build_statuses.png?raw=true)    
+Build Statuses (Dark theme) - Positive, Negative, Unknown.    
 
-## Builds
-Count of builds per week per project.
+![Build Statuses (Light theme)](https://github.com/platform-platform/monorepo/blob/update_project_metrics_document/docs/images/build_statuses_light.png?raw=true)    
+Build Statuses (Light theme) - Positive, Negative, Unknown.    
 
-## Stability
-Percentage of successful builds to total builds per week. For example, if 6 our of 10 builds are successful metric value is 60%. 
+## Build Results Metric
+#### Description
+A build results metric includes the build results and duration of the latest builds. Same as build statuses, build results can be successful, failed, unknown.
+#### Source
+The information about build results comes from the CI system that performs these builds.
+#### Date ranges
+The build results metric includes the results of the last 20 builds
+#### Appearance
+The build results metric appears as a bar graph. Each bar stands for one build result. The height of each bar stands for the build duration.
```

Comment:
Would it be great to mention that the height of each bar stands for the build duration relative to other builds on this graph?